geoportailv3/static/js/draw/drawnfeaturesservice.js

```diff
@@ -179,7 +198,20 @@ app.DrawnFeatures.prototype.saveFeatureInMymaps_ = function(feature) {
 
 
 /**
- * clear
+ * @param {Array.<ol.Feature>} features An array of features to save in mymaps.
+ * @return {angular.$q.Promise} Promise.
+ * @private
+ */
+app.DrawnFeatures.prototype.saveFeaturesInMymaps_ = function(features) {
+  return this.appMymaps_.saveFeatures(features).then(
+      goog.bind(function(resp) {
+        return resp;
+      }, this));
```

Comment:
Same comment as before. Why not directly returning the promise returned by `saveFeatures`?